website/docs/contribute/website.mdx

```diff
@@ -216,7 +216,73 @@ pre-commit run --all-files # double check for any formatting issues not resolved
 git commit -m "foo bar baz"
 git push
 ```
-## Adding images to the website
+
+## Adding to the website
+
+
+### Adding pages
+
+The pages are defined in a file called `website/sidebars.js`
+Depending on what you are adding you can choose to add it into the docs section or examples/ wherever would be most appropriate.
+The pattern and structure in `sidebars.js` should be clear.
+
+For example :
+
+If you want to add another section in between Dashboard and Debug Cable it will be something like this:
```

Comment:
Highlight these somehow, maybe like:
```suggestion
If you want to add another section in between `Dashboard` and `Debug Cable` it will be something like this:
```